src/app/model.ts

```diff
@@ -0,0 +1,33 @@
+import {Tickets, CreatePayload} from 'node-zendesk';
+
+import store from '../store/config';
+
+type CreateTicketFormValues = {
+    subject: string;
+    type: string;
+    priority: string;
+    additional_message: string;
+    post_message: string;
+}
+
+export const getTicketForPost = (values: CreateTicketFormValues): CreatePayload => {
+    const mmSignature = '*message created from Mattermost message.*\n' + store.getSiteURL();
+
+    const zdMessage = values.additional_message + '\n' +
+            values.post_message + '\n' +
+            mmSignature;
```

Comment:
Might want to check if `additional_message` and `post_message` are at least defined here before concatenating strings. Something like

```js
const additionalMessage = values.additional_message || '';
```

This goes for anything else coming from the UI too. We may want some validation here of all fields coming from the `values` object.